lib/console.js

```diff
@@ -249,6 +252,77 @@ Console.prototype.groupEnd = function groupEnd() {
     this[kGroupIndent].slice(0, this[kGroupIndent].length - 2);
 };
 
+const untablableTypes = [
+  'symbol', 'undefined', 'boolean', 'number', 'string',
+];
+
+const untablableValues = [undefined, null, NaN, true, false];
```

Comment:
The booleans, undefined and NaN are actually redundant to the `untablableTypes`. So it would be best to remove the `untablableValues` and just explicitely test for `null` in the `untableable` function.